app/src/main/java/com/duckduckgo/app/tabs/model/TabDataRepository.kt

```diff
@@ -40,11 +42,17 @@ class TabDataRepository @Inject constructor(
     private val siteFactory: SiteFactory,
     private val webViewPreviewPersister: WebViewPreviewPersister,
     private val faviconManager: FaviconManager,
-    private val useOurAppDetector: UseOurAppDetector
+    private val useOurAppDetector: UseOurAppDetector,
+    @AppCoroutineScope private val appCoroutineScope: CoroutineScope
 ) : TabRepository {
 
     override val liveTabs: LiveData<List<TabEntity>> = tabsDao.liveTabs()
 
+    // We only one the new emissions when subscribing, however Room does not honour that contract so we
```

Comment:
there's a typo in the comment, I guess `one` -> `want`?

Reply:
thanks, will fix it!